recipe/spyder-menu.json

```
@@ -109,103 +54,7 @@
-                                "com.apple.xcode.pascal-source",
-                                "com.apple.xcode.strings-text",
-                                "com.apple.xcode.tcsh-script",
-                                "com.apple.xcode.yacc-source",
-                                "com.apple.xcode.zsh-script",
-                                "com.apple.xml-property-list",
-                                "com.netscape.javascript-source",
-                                "com.scenarist.closed-caption",
-                                "com.sun.java-source",
-                                "com.sun.java-web-start",
-                                "net.daringfireball.markdown",
-                                "org.khronos.glsl-source",
-                                "org.oasis-open.xliff",
-                                "public.ada-source",
-                                "public.assembly-source",
-                                "public.bash-script",
-                                "public.c-header",
-                                "public.c-plus-plus-header",
-                                "public.c-plus-plus-source",
-                                "public.c-source",
-                                "public.case-insensitive-text",
-                                "public.comma-separated-values-text",
-                                "public.csh-script",
-                                "public.css",
-                                "public.delimited-values-text",
-                                "public.dylan-source",
-                                "public.filename-extension",
-                                "public.fortran-77-source",
-                                "public.fortran-90-source",
-                                "public.fortran-95-source",
-                                "public.fortran-source",
-                                "public.html",
-                                "public.json",
-                                "public.ksh-script",
-                                "public.lex-source",
-                                "public.log",
-                                "public.m3u-playlist",
-                                "public.make-source",
-                                "public.mig-source",
-                                "public.mime-type",
-                                "public.module-map",
-                                "public.nasm-assembly-source",
-                                "public.objective-c-plus-plus-source",
-                                "public.objective-c-source",
-                                "public.opencl-source",
-                                "public.pascal-source",
-                                "public.patch-file",
-                                "public.perl-script",
-                                "public.php-script",
-                                "public.plain-text",
                                 "public.python-script",
```

```suggestion
                                "public.python-script"
```
Final list item with "," results in `JSONDecodeError`